alb_no_logs.tf

```diff
@@ -16,7 +16,7 @@ resource "aws_lb" "application_no_logs" {
     update = "${var.load_balancer_update_timeout}"
   }
 
-  count = "${var.logging_enabled ? 0 : 1}"
+  count = "${var.logging_enabled && !var.create_alb ? 0 : 1}"
```

Comment:
You can introduce a new `local` value like `create_alb_with_logging_enabled` and compare it in `count`.